trusted-types/set-attributes-mutations-in-callback.tentative.html

```diff
@@ -19,8 +19,9 @@
     if (window.applyMutation) {
       assert_equals(input, input_string);
       window.applyMutation();
+      return output_string;
     }
-    return output_string;
+    return input;
```

Comment:
Mmh, probably we should keep the function as is and only use it for the default policy. Probably renaming it `applyDefaultPolicyMaybeApplyingMutation` or something like that.

We can have a separate policy that just returns the input, for the sake of creating trusted types.